lib/plugins/aws/deploy/lib/uploadArtifacts.test.js

```diff
@@ -179,7 +179,7 @@ describe('uploadArtifacts', () => {
         expect(readFileSyncStub.calledOnce).to.equal(true);
         expect(putObjectStub.calledWithExactly(
```

Comment:
@pmuens Maybe the stub itself can be named to match the called function too.

Reply:
Nice catch! Thanks @HyperBrain 👍

Just updated the stub names!